websites/A/Animes Portal/iframe.ts

```diff
@@ -0,0 +1,11 @@
+const iframe = new iFrame();
+iframe.on("UpdateData", async () => {
+  const video = document.querySelector<HTMLVideoElement>("#html5player");
+  if (video && !isNaN(video.duration)) {
```

Comment:
learn how optional chaining works
```suggestion
  if (!isNaN(video?.duration)) {
```

Reply:
Not sure if that works in isNaN since null is not NaN

Reply:
It'll be `undefined` not `null`, so it would work